voc/voc.go

```diff
@@ -42,6 +42,11 @@ type IsCloudResource interface {
 	GetType() []string
 	HasType(string) bool
 	GetCreationTime() *time.Time
+	Related() []string
```

Comment:
Thus, every resource has to implement Related. Is it done?

Reply:
The `Resource` which is embedded in each resource has a default implementation that returns an empty string.